crates/forge/test_utils/src/runner.rs

```diff
@@ -300,6 +300,16 @@ pub fn assert_gas(result: &[TestTargetSummary], test_case_name: &str, asserted_g
     }));
 }
 
+// This logic is used to assert exact gas values in CI for the minimal supported Scarb version
+// and to assert gas values with a margin in scheduled tests, as values can vary for different Scarb versions
```

Comment:
Let's add something "scary" to it like `FOR LOCAL DEVELOPMENT ALWAYS USE EXACT CALCULATIONS`